src/gradualizer.erl

```diff
@@ -13,7 +13,8 @@
          type_check_module/1,
          type_check_module/2,
          type_check_dir/1,
-         type_check_dir/2
+         type_check_dir/2,
+         get_forms_from_beam/1
```

Comment:
If this function must be exported, I think it should be be moved to another module. In my opinion, it is also OK to duplicate some lines from this function. The case when the beam file doesn't contain abstract code is not an error in this case, but just a case where we should try the fallback instead.